<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIChangePswd</class>
 <widget class="QWidget" name="UIChangePswd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QLabel" name="CgPswdImg">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>351</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string>IMAGE</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widet" native="true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>421</width>
     <height>601</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="textUserName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 13px</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textUserID">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 13px</string>
    </property>
    <property name="maxLength">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="textUserOldPswd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 13px</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
    <property name="text">
     <string>学号/工号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 12px</string>
    </property>
    <property name="text">
     <string>原密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>方正黑体简体</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16px &quot;方正黑体简体&quot;;</string>
    </property>
    <property name="text">
     <string>请按提示输入下列信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSure">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*正常样式*/
QPushButton
{	background-color: rgb(29, 123, 255);
	color: rgb(255, 255, 255);
	font: 25 12px &quot;Bahnschrift Light&quot;;
}

/*鼠标停留样式*/
QPushButton:hover
{
	font: 25 12px &quot;Bahnschrift Light&quot;;
	background-color: #4090ff;
}</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonReturn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">/*正常样式*/
QPushButton
{
	background-color: transparent;
	color: #1c7eff;
	font: 25 12px &quot;Bahnschrift Light&quot;;
	border-style:solid
}

/*鼠标停留样式*/
QPushButton:hover
{
	color: #21aeff;
	font: 25 12px &quot;Bahnschrift Light&quot;;
}</string>
    </property>
    <property name="text">
     <string>Click to return Login</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="textUserPswd_1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 13px</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入新密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textUserPswd_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Candara</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 13px</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请再次输入新密码</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textUserName</tabstop>
  <tabstop>textUserID</tabstop>
  <tabstop>textUserOldPswd</tabstop>
  <tabstop>textUserPswd_1</tabstop>
  <tabstop>textUserPswd_2</tabstop>
  <tabstop>ButtonSure</tabstop>
  <tabstop>ButtonReturn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
